test/integration/confirmations/signatures/signature-helpers.ts

```diff
@@ -91,6 +91,51 @@ export const getMetaMaskStateWithUnapprovedPermitSign = (
   };
 };
 
+export const getMetamaskStateWithMaliciousPermit = (accountAddress: string) => {
+  const pendingPermitId = '48a75190-45ca-11ef-9001-f3886ec2397c';
```

Comment:
[minor nit] we can move this out to avoid reinstantiating for each getMetamaskStateWithMaliciousPermit call 
```suggestion  
const pendingPermitId = '48a75190-45ca-11ef-9001-f3886ec2397c';

export const getMetamaskStateWithMaliciousPermit = (accountAddress: string) => {
```

Reply:
done